esteban-madrigal-ak/training-assignments
# Task01 html css

# Pull Request
## Description
This branch was used to develop a character viewer feature, there are just 2 files, the html and the css that allow to show 4 images and the names of 4 characters.

## Type of change

## How Has This Been Tested?
I test the functionality of the application with help of an extension called live server that helps to see the changes of the html and how it looks.
1. install the extension
2. open the html file you want to test on vsc
3. select the go live option on the bottom panel
4. it will open the web browser showing the html
5. every change on the html or css is going to be showed on the browser window
